test/jit/test_freezing.py

```diff
@@ -806,9 +806,6 @@ def forward(self, x):
         expected = m_s.forward(inp)
         self.assertEqual(out, expected)
 
-    # Check attribute a is preserved. Alias analysis detects that 'a' has output writers.
-    # In this example, 'a' is not mutated. However, we do not track which sub
-    # values of a composite ivalue is mutated.
     def test_freeze_module_with_aliased_attr2(self):
```

Comment:
This test is actually asserting that we *can't*  track aliasing inside tuples. Since we can and should do so, removing this assert.